src/TensorFlow.jl

```diff
@@ -165,14 +165,13 @@ library loaded.
 macro py_proc(expr)
     quote
         eval(Main, quote
-            remotecall_fetch($(TensorFlow.load_python_process())) do
+            remotecall_wait($(TensorFlow.load_python_process())) do
```

Comment:
It's because when a `PyObject` is fetched, the pointer in it is zeroed-out since Julia doesn't want you passing pointers between processes. But you can safely return a future to the pointer with `wait`, which you can then reference in a subsequent call to the Python process.